test/preflight/apps_v2_integration_test.go

```diff
@@ -678,7 +678,7 @@ func TestAppsV2MigrateToV2_Volumes(t *testing.T) {
 	// No spaces or quotes, this is sent unescaped to bash :x
 	successStr := "myvolumehasloaded"
 
-	f.Fly("launch --org %s --name %s --region %s --internal-port 80 --force-nomad --image nginx", f.OrgSlug(), appName, f.PrimaryRegion())
+	f.Fly("launch --org %s --name %s --region %s --internal-port 80 --force-nomad --image nginx --ha=false", f.OrgSlug(), appName, f.PrimaryRegion())
```

Comment:
hmm.. this is weird because `--ha=false` doesn't change deploy behavior for nomad apps

Reply:
oh yeah you're totally right